Tests/SwinjectAutoregistrationTests/TypeParserTests.swift

```diff
@@ -129,7 +129,7 @@ class TypeParserSpec: QuickSpec {
             }
             
             it("scans implicitly unwrapped optional") {
-                let string = "\((String!).self)"
+                let string = "\((String?).self)"
```

Comment:
This test should be removed, since the same case is covered in the below test case. I've done so on my PR #46 .